### DISCUSSION TITLE
`tts` command freeze after 'model already downloaded'

### DISCUSSION THREAD

**Discussion by yankeguo** [coqui-ai/TTS#3205] [General Q&A]
`tts` command freeze after displaying `> tts_models/multilingual/multi-dataset/xtts_v2 is already downloaded.`, for at least few minutes.

`Ctrl+C` will unfreeze the command, and everything continues fine.

I don't see any CPU/MEM or GPU usage during freeze, I don't know what happened.

![截屏2023-11-13 14 38 42](https://github.com/coqui-ai/TTS/assets/52509957/7263a765-a0d4-4850-b6fd-af8f28c7e190)

Screenshot provided, check the position of `^C` mark.

**Comment by xufengnian** [comment]
same question

**Reply by yankeguo** [reply]
I switched to manual invocation, see https://tts.readthedocs.io/en/latest/models/xtts.html#model-directly

(`deepspeed` is not mandatory)

**Reply by xufengnian** [reply]
you answer help me a lot,thank you!
我猜你也是中国人，我就用中文了：老铁666！

**Reply by yankeguo** [reply]
还有一个方法，屏蔽掉一个函数就行

```python
from TTS.utils.manage import ModelManager

def no_check(*args, **kwargs):
    pass

ModelManager.check_if_configs_are_equal = no_check
```

**Reply by xufengnian** [reply]
老哥，你发现了问题的本质，这个做法的通用性和便利性更强！啥也不说了，牛逼！

**Comment by yankeguo** [comment]
Looks like it's stuck here due to local network issues

https://github.com/coqui-ai/TTS/blob/04901fb2e4b74953bb5733205adcb7cb13655a06/TTS/utils/manage.py#L355
